library/src/main/java/com/novoda/downloadmanager/lib/DownloadInfo.java

```diff
@@ -217,7 +239,10 @@ private Long getLong(String column) {
     public int mBypassRecommendedSizeLimit;
     public String bigPictureResourceUrl;
 
-    public int mFuzz;
+    public Uri mAllDownloadsUri;
```

Comment:
if you're creating new fields, use Novoda coding standards - we are moving away from the Google code and this helps highlight what we have added or changed.